package.json

```diff
@@ -44,8 +44,9 @@
     "danger": "danger ci",
     "prettier": "prettier --config .prettierrc --write \"**/*.js\"",
     "create-component": "node scripts/npm/create-component.js",
+    "submodules:reset": "git checkout origin/master backpack-android backpack-ios backpack-react-native && git submodule update --init --recursive",
     "submodules:pull": "git submodule foreach git pull origin master",
-    "submodules:update": "npm run submodules:unlock && npm run submodules:pull && git add backpack-android && git add backpack-ios && git add backpack-react-native && git commit -m\"[skip ci] Update submodules\" && npm run submodules:lock",
+    "submodules:update": "npm run submodules:unlock && npm run submodules:pull && git add backpack-android backpack-ios backpack-react-native && git commit -m\"[skip ci] Update submodules\" && npm run submodules:lock",
```

Comment:
Also, `git add` accepts any number of path args, so no need to call the command 3 times.